core/prometheus_exporter.go

```diff
@@ -0,0 +1,131 @@
+package core
+
+import (
+	"fmt"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	namespace = "gorb" // For Prometheus metrics.
+)
+
+var (
+	serviceHealth = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: namespace,
+		Name:      "service_health",
+		Help:      "Health of the load balancer service",
+	}, []string{"name", "host", "port", "protocol", "method", "persistent"})
```

Comment:
My understanding of labels is that they are meant to disambiguate metrics. If that's the case, then I think "method" and "persistent" shouldbn't be labels since there cannot be two services with different LB methods or different settings for persistance but same everything else.